subworkflows/local/mirtrace.nf

```diff
@@ -15,7 +15,7 @@ workflow MIRTRACE {
     ch_mirtrace_config =
     reads.map { adapter, ids, reads -> [ids,reads]}
     .transpose()
-    .collectFile { id, path -> "./${path.getFileName().toString()}\n" } // operations need a channel, so, should be outside the module
+    .collectFile { id, path -> "./${path.getFileName().toString()},${params.phred_offset}\n" } // operations need a channel, so, should be outside the module
```

Comment:
According to the docs PHRED is only the last of four columns: 
![image](https://github.com/nf-core/smrnaseq/assets/7051479/2bddedca-bd10-466a-b90c-89621fa5184a)

Reply:
![image](https://github.com/nf-core/smrnaseq/assets/7051479/f7a1ef6a-5ff4-444e-881c-5d7619314935)

Reply:
Adjusted it